post_installation_configuration/coreos-layering.adoc

```diff
@@ -82,19 +146,45 @@ Because libreswan requires additional RHEL packages, the image must be built on
 ** SSH Key management packages.
 --
 +
-.Example Containerfile to apply a third-party package from EPEL
+.Example on-cluster Containerfile to apply a third-party package from EPEL
 [source,yaml]
 ----
-include::https://raw.githubusercontent.com/openshift/rhcos-image-layering-examples/master/htop/Containerfile[]
+# Get RHCOS base image of target cluster `oc adm release info --image-for rhel-coreos`
```

Comment:
issue: They wouldn't need to get the base image pullspec if they're doing this on-cluster since the on-cluster layering process will inject it for them. All they need to provide is the `FROM configs AS final` line.